http-svr/xmlrpc_client.ml

```diff
@@ -206,6 +206,8 @@ let with_reusable_stunnel ?use_fork_exec_helper ?write_to_log ?verify_cert host
         StunnelDebug.error "get_reusable_stunnel: failed to acquire a working stunnel to connect to %s:%d" host port;
         raise Stunnel_connection_failed
   in loop ()
+(* supress warning about unused variables for the verify_cert parameter *)
+[@@ocaml.warning "-27"]
```

Comment:
Wouldn't it be better to delete the `verify_cert` parameter then?
Having it there but not taking effect is confusing. I see there is a global now that controls whether certs are verified or not.

Reply:
I'll check users of the functions.

Reply:
Yes, you might have to update XAPI, if you haven't already

Reply:
xen-api needs some changes to deprecate a parameter and remove this usage.